src/slash-bedrock/share/brl-fetch/distros/manjaro

```diff
@@ -0,0 +1,133 @@
+#!/bedrock/libexec/busybox sh
+#
+# Manjaro Linux bootstrap support
+#
+#      This program is free software; you can redistribute it and/or
+#      modify it under the terms of the GNU General Public License
+#      version 2 as published by the Free Software Foundation.
+#
+# Copyright (c) 2016-2019 Daniel Thau <danthau@bedrocklinux.org>
+#
+
+# shellcheck source=src/slash-bedrock/libexec/brl-fetch
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+	true
```

Comment:
No problem, I will update this. I had it set to ```true``` so that I could brl fetch without any additional options when I was testing.
As for supporting it, yes, I will be supporting it -- but you can decide if / when to turn it into a supported distro.

Reply:
> No problem, I will update this. I had it set to true so that I could brl fetch without any additional options when I was testing.

`true`/`false` here doesn't change options required when testing.  It just changes if it shows up when one runs `brl fetch --list` versus when one runs `brl fetch --experimental`.

> As for supporting it, yes, I will be supporting it -- but you can decide if / when to turn it into a supported distro.

This doesn't parse for me.  There shouldn't be a disconnect between whether someone is officially supporting Manjaro usage in Bedrock and whether or not Manjaro is officially documented as supported in Bedrock.  It's not something I decide arbitrarily.  Maybe I'm confused by what you mean by "supporting" and what you mean by "it."

The concern with `check_supported()` is whether or not an adequately capable person is actively ensuring all notable distro-specific bits work as part of a Bedrock Linux system.  If there is, we document it [here](https://bedrocklinux.org/0.7/distro-support.html), make `check_supported()` `true`, and direct any user reported issues to the corresponding maintainer.  Users should be able to rely on this to a reasonable degree.  If not, we mark it N/A [here](https://bedrocklinux.org/0.7/distro-support.html), make `check_supported()` `false`, and apologies to users who have issues and explain that we don't have anyone with the expertise and bandwidth to remedy whatever problems they're having.

If:

- You know Manjaro very well (e.g. I've been using Debian for over a decade, so I can support Debian.  Philantrop is on the Exherbo team, so he can support Exherbo).
- You regularly, actively use Manjaro as part of a Bedrock Linux system on a day-to-day basis such that you're likely to personally run into Bedrock/Manjaro issues once they arise.
- You are reasonable active in some Bedrock communication platform (IRC, reddit, github, LQ forums) such that people who need assistance can get your attention in short order.
- You are willing and eager to undertake the associated responsibility.

we can add you to [this table](https://bedrocklinux.org/0.7/distro-support.html) and make `check_supported()` here `true`.  Otherwise, that column is `N/A` and `check_suppored()` here should remain `false` until such a person shows up.